etc/gummi.profile

```diff
@@ -0,0 +1,18 @@
+# Firejail profile for gummi
+# This file is overwritten after every install/update
+# Persistent local customizations
+include gummi.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/gummi
```

Comment:
`${HOME}/.cache/gummi` (.cache has ist own section in disable-programs.inc (end of file))

Reply:
Thanks. I was not able to check for special directories.

Running ```$ firejail --join=gummi ls -aR``` after ```firejail --name=gummi --ignore=private-bin --profile=/etc/firejail/gummi.profile --private /usr/bin/gummi``` does not work for me.  
I just see the contents of the current dir.

As I have worked on Gummi myself, I assumed I'd have remembered all relevant directories.